doc/LICENSE.txt

```diff
@@ -9,9 +9,11 @@ All rights reserved.
 Contains code from NumPy, Copyright (c) 2005-2016, NumPy Developers.
 All rights reserved.
 
-Contain CnMeM under the same license with this copyright:
+Contains CnMeM under the same license with this copyright:
 Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
 
+Contains frozendict code from slezica’s python-frozendict(https://github.com/slezica/python-frozendict/blob/master/frozendict/__init__.py), Copyright (c) 2012 Santiago Lezica. All rights reserved. 
```

Comment:
Good idea to also add it here. But you need to use the license information from that code. It is not all rights reserved.

```
Contains frozendict code from slezica’s python-frozendict(https://github.com/slezica/python-frozendict/blob/master/frozendict/__init__.py), MIT License. See the top of the file theano/misc/frozendict.py for detail.
```